pallets/allocations/src/tests.rs

```diff
@@ -141,7 +154,14 @@ type Events = pallet_allocations::Event<Test>;
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	sp_tracing::try_init_simple();
 
-	let storage = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
+	let mut storage = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
+
+	pallet_membership::GenesisConfig::<Test> {
+		members: vec![Oracle::get()],
+		..Default::default()
+	}
+	.assimilate_storage(&mut storage)
+	.unwrap();
```

Comment:
Even though this is in the test code `unwrap()` should still be avoided, it could remove some useful clues to why a test case is failing.